test/fixtures/programmatic/curated-plugins/serverless.yml

```diff
@@ -2,6 +2,7 @@ service: service
 
 configValidationMode: error
 frameworkVersion: '*'
+useDotenv: true
```

Comment:
Should that be added? I think property will enable our internal logic that loads env vars instead of using the plugin

Reply:
This doesn't change how plugin works, and the only difference is that given env variable will be loaded earlier in a process (otherwise it'll be loaded at plugin initialization).

I've added this, as tests should not trigger any deprections (otherwise they will crash). Technically I could also achieve that by disabling the specific deprecation, but thought that not triggering deprecation is neater (reflects what we expect from users)

Reply:
Okay, I was just wondering if what you wanted to also test is that the plugin loads the env variables vs Framework internals doing that work, but I think the approach here is fine as it's still testing the extra capabilities of plugin. :+1: